src/swarm/neo/client/mixins/Controllers.d

```diff
@@ -323,3 +325,21 @@ template Controllers ( )
         }
     }
 }
+
+/// Unit test instantiating the templates to detect semantic errors
+version (UnitTest) private class ControllersTemplateTest
```

Comment:
Any idea why the test client didn't catch this?: https://github.com/sociomantic-tsunami/swarm/blob/v4.x.x/integrationtest/neo/client/Client.d#L264-L270

Reply:
That code seems to use [`SuspendableController`](https://github.com/sociomantic-tsunami/swarm/blob/v4.x.x/src/swarm/neo/client/mixins/SuspendableRequestCore.d#L513).

Reply:
That's the internal controller implementation inside a client request handler. It may use that, but it's a different thing.

Reply:
I just looked into this a bit more. I was misunderstanding. The module that you've added the test for is intended for use in application code. The test client that I was referring to doesn't use this.